tweets/node-12-eol.tweet

```diff
@@ -0,0 +1,5 @@
+FYI: Node.js 12.x went EOL 🪦 on April 30, 2022.
```

Comment:
12.x to 12.
```suggestion
FYI: Node.js 12 went EOL 🪦 on April 30, 2022.
```

Similar to https://twitter.com/nodejs/status/1388116425361874945